packages/forms/src/fields/ObjectField.js

```diff
@@ -103,6 +105,9 @@ class ObjectField extends Component {
 		const { widget, ...options } = getUiOptions(uiSchema);
 		// widget
 		if (typeof widget === 'string') {
+			if (widget === 'hidden') {
+				return (<HiddenWidget />);
```

Comment:
Why do you return input here? Please, try to render nothing!
The goal is to hide this object :)